pkg/karmadactl/get_flags.go

```diff
@@ -0,0 +1,437 @@
+package karmadactl
+
+/*
```

Comment:
It seems that this file was copied from k8s.io/cli-runtime. Do we have a modification? Could we reuse vendor files instead of copying again?

Reply:
In this code, I have modified the toRawKubeConfigLoader func. Used to regenerate the flags used to create the Factory.
k8s.io/cli-runtime it doesn't meet my needs, so I simply modified the code.

Reply:
lint error, because our vendor directory has cover.out, but .gitignore has *.out, so I can't upload to github.

Reply:
Here shows a solution.
https://stackoverflow.com/questions/8006393/force-add-despite-the-gitignore-file